recipes/namedtype/all/conanfile.py

```diff
@@ -17,10 +20,37 @@ def _source_subfolder(self):
 
     def source(self):
         tools.get(**self.conan_data["sources"][self.version])
-        url = self.conan_data["sources"][self.version]["url"]
-        extracted_dir = "NamedType-" + os.path.splitext(os.path.basename(url))[0]
+        extracted_dir = glob.glob("NamedType-*")[0]
         os.rename(extracted_dir, self._source_subfolder)
 
+    @property
+    def _minimum_cpp_standard(self):
+        return 11
+
+    @property
+    def _minimum_compilers_version(self):
+        return {
+            "Visual Studio": "15",
+            "gcc": "5",
+            "clang": "3.9",
+            "apple-clang": "8",
+        }
```

Comment:
minimum compiler versions seems very high for C++11, why not dropping this part like others recipes requiring C++11?